py_modules/user_api.py

```diff
@@ -17,23 +17,18 @@ def check_if_user_exists():
         # if user isn't logged -> start the "subscription app" the first time
         try:
             # Creating the sqlite3 database
-            DBApi.create_database()
-
```

Comment:
Il database viene automaticamente creato? Questa istruzione era stata aggiunta per evitare che il programma andasse in exception se il db non esisteva!

Reply:
Si, vengono eseguite tutte le migrazioni già alla prima volta che si importa il modulo. Quindi verrà eseguito tutto prima ancora che l'utente possa utilizzare una qualunque istanza di DBApi.